metabase/metabase
# [Dashboard] Make apply button aligned with fields on small screens

Closes https://github.com/metabase/metabase/issues/34385

### Description

Fix button position on dashboard with auto-apply filters disabled on small screens.
Button has the same height as filter and full width

### How to verify

- create a dash, add filters there
- disabled auto-apply filters (click 'i' on the dashboard screen, toggle)
- enable small screen emulator  and change a value of some filter -> Apply button should appear


### Demo

![image (8)](https://github.com/metabase/metabase/assets/125459446/748fc19f-3724-47bc-a44e-1510f12c280c)

<img width="472" alt="Screenshot 2023-10-25 at 19 14 53" src="https://github.com/metabase/metabase/assets/125459446/5a61dec1-2d54-43a5-9a31-4c9847bc895a">
<img width="429" alt="Screenshot 2023-10-25 at 19 14 50" src="https://github.com/metabase/metabase/assets/125459446/c0d76cef-65dd-4a5c-8f90-1c987c7738a1">

### Checklist

~- [ ] Tests have been added/updated to cover changes in this PR~
